force-app/main/default/objects/Metadata_Driven_Trigger__mdt/fields/Execution_Order__c.field-meta.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Execution_Order__c</fullName>
+    <externalId>false</externalId>
+    <fieldManageability>DeveloperControlled</fieldManageability>
```

Comment:
I think it is better to make this `SubscriberControlled`

Reply:
since it's not in a namespace/package I don't know that I see a difference? Say more please

Reply:
it's up to you! Since I come building a lot of Managed package apps I always consider the possibility of someone taking this code and packaging it.

Also, the SubscriberControlled type fits better since we want admins to manage it or users with proper perms to manage it.

Feel free to ignore it :) It was just a nitpick!